rpm/droidmedia.spec

```diff
@@ -8,7 +8,7 @@
 
 Name:          droidmedia
 Summary:       Android media wrapper library
-Version:       0.0.0
+Version:       0.20170214.0
```

Comment:
No it still get replaced by OBS, it just satisfies a requirement in gst-droid, so it's useful for local builds. I left it in because someone else might find it useful, and it's more correct than 0.0.0.